app/views/devise/registrations/edit.html.haml

```diff
@@ -15,7 +15,10 @@
           %h4
             Currently the following openIDs are associated with your account
           - @openids.each do |openid|
-            %li= "#{openid.provider}:#{openid.email}"
+            %li
+              %span "#{openid.provider}:#{openid.email}"
+              = link_to user_openid_path(openid, user_id: current_user.id), method: :delete do
```

Comment:
We should probably show a confirmation message first, before proceeding to the deletion.